src/Set.mo

```diff
@@ -1777,7 +1749,7 @@ module {
           case (#deleted) { #deleted };
           case (#inexistent) { #inexistent };
           case (#mergeChild({ internalChild })) {
-            #mergeChild({ internalChild; deletedElement = deleteElement })
+            #mergeChild({ internalChild })
```

Comment:
(Fixed a compiler warning)

Reply:
Was this just a spurious, unexpected record field caught by Gabor's new warning?